modules/tinymce/src/plugins/table/test/ts/module/test/TableModifiersTestUtils.ts

```diff
@@ -78,64 +78,80 @@ const assertStructureHasCustomStyle = (editor: Editor, rows: number, columns: nu
     '</table>');
 };
 
-const closeMenu = (editor) =>
+const closeMenu = (editor) => {
   TinyUiActions.keydown(editor, Keys.escape());
+  TinyUiActions.keydown(editor, Keys.escape());
+};
 
-const openMenu = (editor: Editor, title: string) =>
-  TinyUiActions.clickOnToolbar(editor, `button[title="${title}"]`);
+const clickOnButton = async (editor: Editor, title: string, toolbar: boolean) => {
+  if (toolbar) {
+    TinyUiActions.clickOnToolbar(editor, `button[title="${title}"]`);
+  } else {
+    TinyUiActions.clickOnMenu(editor, 'span:contains("Table")');
+    await TinyUiActions.pWaitForUi(editor, `div[title="${title}"]`);
+    TinyUiActions.clickOnUi(editor, `div[title="${title}"]`);
+  }
+};
 
-const pAssertMenuPresence = async (editor: Editor, label: string, menuTitle: string, expected: Record<string, number>, container: SugarElement<HTMLElement>) => {
-  openMenu(editor, menuTitle);
+const pAssertMenuPresence = async (editor: Editor, label: string, menuTitle: string, expected: Record<string, number>, container: SugarElement<HTMLElement>, toolbar: boolean) => {
+  await clickOnButton(editor, menuTitle, toolbar);
```

Comment:
As this is being overloaded and it really shouldn't be, you'll need to either pass in an open function or just do it separately before calling this.

- [ ] Don't overload functions like this please, keep it specific to a single action